pubnub/pubnub_asyncio.py

```diff
@@ -591,7 +591,7 @@ def adapt_subscribe_builder(self, subscribe_operation: SubscribeOperation):
                 with_presence=subscribe_operation.presence_enabled
             )
         self.event_engine.trigger(subscription_event)
-        if self._pubnub.config._heartbeat_interval > 0:
+        if self._pubnub.config.enable_presence_heartbeat and self._pubnub.config._heartbeat_interval > 0:
```

Comment:
I'm curious, why are 2 flags for heartbeat interval like `enable_presence_heartbeat` and `heartbeat_interval`?

Reply:
by default `self.enable_presence_heartbeat = False` so previously you could have it set to `False` and still working heartbeat loop ;)

Reply:
What I meant is that having `heartbeat_interval` should be enough to determine whether sending heartbeat request should take place or not. How long `enable_presence_heartbeat` has been present in this SDK?